infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -118,6 +118,10 @@ public void specificationFailure(final String description, final Throwable cause
     log.warn("Spec failed for {}: {}", description, cause, cause);
   }
 
+  public void failedToLoadValidatorKey(final String message) {
```

Comment:
I'd rename it to `failedToLoadValidatorKeys` or `failedToLoadValidatorKeyStore` since we're talking about all the keys

Reply:
it's literally 1 key, so for me i'm ignoring this one.